docs/install-upgrade/build-wiring.md

```diff
@@ -56,28 +54,33 @@ OPTIONS:
 ```
 
 ### Sample Switch Configuration
-```yaml
+``` { .yaml .annotate linenums="1" }
 apiVersion: wiring.githedgehog.com/v1beta1
 kind: Switch
 metadata:
   name: ds3000-02
 spec:
-  boot:
-    serial: ABC123XYZ
+  boot: # Serial or MAC can be used
+    serial: ABC123XYZ # alternative  `mac: 34:AD:61:00:02:03` of 1GbE interface
```

Comment:
I think there is no need to put mac in the comment, just put it into a spec like serial. Additionally, 1GbE isn't very correct - it should be first management port as there could be multiple 1GbE/management ports

Reply:
basically suggestion from https://github.com/githedgehog/docs/pull/77#discussion_r2010522709

Reply:
Done!